charts/prl-ccd-definitions/values.preview.template.yaml

```diff
@@ -123,7 +123,7 @@ ccd:
         OIDC_ISSUER: https://forgerock-am.service.core-compute-idam-aat.internal:8443/openam/oauth2/hmcts
         ELASTIC_SEARCH_ENABLED: true
         ELASTIC_SEARCH_HOST: ${SERVICE_NAME}-es-master
-        ELASTIC_SEARCH_CASE_INDEX_MAPPING_FIELDS_LIMIT: 15000
+        ELASTIC_SEARCH_CASE_INDEX_MAPPING_FIELDS_LIMIT: 16000
```

Comment:
is this change intended for this PR?

Reply:
this limit change currently applicable only for this pr. But once this has been merged into Release-3-1, i am not sure whether we need to keep the same limit. Can you please confirm me on this.